src/actions/actions.js

```diff
@@ -269,6 +269,13 @@ function getNotebookSaveRequestOptions(state) {
     title: state.title,
     content: exportJsmdToString(state),
   }
+  const revisionID = state.notebookInfo.revision_id
+  const userOwns = state.notebookInfo.user_can_save
+  const forkedFrom = revisionID || undefined
+  // only set forked_from if user doesn't own this notebook.
```

Comment:
Hmm, I feel like we might want to keep track of the history even if the user is forking their own notebook?

Reply:
good point

Reply:
Going to generalize this a little bit so we can make it easy for a user to fork their notebook from a menu item, etc.